superbench/benchmarks/model_benchmarks/pytorch_bert.py

```diff
@@ -23,7 +28,32 @@ def __init__(self, config, num_classes):
             num_classes (int): The number of objects for classification.
         """
         super().__init__()
-        self._bert = BertModel(config)
+        self._enable_fp8 = False
+        if hasattr(config, 'precision'):
+            self._enable_fp8 = config.precision.startswith('FP8_')
+            delattr(config, 'precision')
```

Comment:
maybe this can be moved into model_base I think? it should be general for all models to support fp8 precision？